src/FSharp.Core/list.fsi

```diff
@@ -1655,6 +1849,9 @@ module List =
 
     /// <summary>Applies the given function to successive elements, returning the first
     /// result where function returns <c>Some(x)</c> for some x. If no such
+    /// 
+    /// Time complexity: O(n) - worst case when element is at the end or not found
+    /// Space complexity: O(1)
```

Comment:
Here the added complexity information should be included at the end of the summary and not in the middle of a sentence